test.md

```diff
@@ -655,9 +667,17 @@ Use the `warning` class to let people know this is dangerous or they should pay
 {: .warning}
 ```
 
-It will render like this  with a colored sidebar and icon:
+The 'warning' class renders as follows:
 
-![warning admonition example](/images/warning-admonition-example.png)
+> **Warning**
+>
+> Removing Volumes
+>
+> By default, named volumes in your compose file are NOT removed when running `docker-compose down`. If you want to
+>remove the volumes, you will need to add the `--volumes` flag.
+>
+>The Docker Dashboard does _not_ remove volumes when you delete the app stack.
```

Comment:
Perhaps wrap this, and add a space between `>` and the text on each line;

```suggestion
> By default, named volumes in your compose file are NOT removed when running
> `docker-compose down`. If you want to remove the volumes, you will need to add
> the `--volumes` flag.
>
> The Docker Dashboard does _not_ remove volumes when you delete the app stack.
```

_However_ when viewing this on GitHub,  users won't see what it looks like when used in the docs (I _guess_ this is why an image was used to show what it looks like)